pkg/jobs/pool.go

```diff
@@ -208,9 +209,18 @@ func (p *Pool) GetJobStatus(id string) string {
 
 //GetStatus - returns the status of the pool of jobs
 func (p *Pool) GetStatus() string {
+	p.poolStatusLock.Lock()
+	defer p.poolStatusLock.Unlock()
 	return p.poolStatus.String()
 }
 
+//SetStatus - Sets the status of the pool of jobs
+func (p *Pool) SetStatus(status PoolStatus) {
+	p.poolStatusLock.Lock()
+	p.poolStatus = status
+	p.poolStatusLock.Unlock()
```

Comment:
why not do the defer here like above?

Reply:
Could. I thought that since it was only one line away it didn't matter. I can change if you wish.

Reply:
I just didn't like that its one way in Get and another in Set...your choice

Reply:
Yeah, I did think about that. I'm changing it.